### PR TITLE
Adding disease page

### PR DESCRIPTION
**Pull request by olaaustine**
[G2P-79](https://www.ebi.ac.uk/panda/jira/browse/G2P-79)

### CODE REVIEW

File: src/views/DiseaseView.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        <div v-if="diseaseData && diseaseSummaryData">
+            <h2 v-if="diseaseData.name">{{ diseaseData.name }} </h2>
+            <h2 v-else class="text-muted">Not Available</h2>
+            <h3 v-for="item in diseaseData.ontology_terms">{{ item.description }}</h3>
+            <!-- v.else is a directive to do if v-if is  not True -->
+            <h4 v-if="diseaseData.mim">
+                <a :href="`https://omim.org/entry/${diseaseData.mim}`" target="_blank">{{ diseaseData.mim }}</a>
+            </h4>
+            <div class="container px-5 py-3">
+                <h4 class="py-3">Records associated with this disease</h4>
```
Comment (seeta-ramaraju) [inline src/views/DiseaseView.vue:73]:
```suggestion
                <h4 class="py-3">Latest Records</h4>
```

File: src/views/DiseaseView.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    <a :href="`http://purl.obolibrary.org/obo/${item.accession.replace(/:/g, '_')}`" target="_blank">{{
+            item.accession }}</a>
+                </p>
+                <h4 v-if="diseaseData.publications && diseaseData.publications.length > 0">Publications</h4>
+                <ul v-for="item in diseaseData.publications">
+                    <li>
+                        <a :href="`https://pubmed.ncbi.nlm.nih.gov/${item.pmid}`" target="_blank">{{ item.title }}</a>
+                    </li>
+                </ul>
+                <p v-if="diseaseData.last_updated && diseaseData.last_updated != ''"> Date updated: {{
```
Comment (seeta-ramaraju) [inline src/views/DiseaseView.vue:133]:
```suggestion
                <p v-if="diseaseData.last_updated && diseaseData.last_updated != ''">Last Update: {{
```

**Review by seeta-ramaraju — COMMENTED** [review]
Rest all code changes looks good to me. Suggested comments to change header texts so that we can have consistent header texts across all other pages.

**Comment by olaaustine** [comment]
> Rest all code changes looks good to me. Suggested comments to change header texts so that we can have consistent header texts across all other pages.

Thank you @seeta-ramaraju, changes applied.

**Review by seeta-ramaraju — APPROVED** [review]
Looks good to me

**Pull request merged**